src/main/java/com/example/repository/AdministratorRepository.java

```diff
@@ -73,8 +74,9 @@ public Administrator findByMailAddressAndPassward(String mailAddress, String pas
 	 * 
 	 * @param administrator 管理者情報
 	 */
-	public void insert(Administrator administrator) {
-		SqlParameterSource param = new BeanPropertySqlParameterSource(administrator);
+	public void insert(Administrator administrator) 
+	{
+	SqlParameterSource param = new BeanPropertySqlParameterSource(administrator);
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 8, expected level should be one of the following: 4, 6.